cmd/api-linter/cli.go

```diff
@@ -104,12 +104,17 @@ func runCLI(rules lint.Rules, configs lint.Configs, args []string) error {
 				switch c.String("fmt") {
 				case "json":
 					marshal = json.Marshal
+				case "summary":
+					marshal = func(i interface{}) ([]byte, error) {
+						summary := createSummary(i.([]lint.Response))
```

Comment:
Is there a need for 2 functions?

How about simply `return emitSummary(i.([]lint.Response))